README.md

```diff
@@ -109,6 +109,17 @@ Build a Docker image for a particular node role (replace `$ROLE` with `collectio
 make docker-native-build-$ROLE
 ```
 
+#### Building a binary for the access node
```

Comment:
This section is being referred [here](https://docs-git-vishal-accessnodesetupupdate-onflow.vercel.app/networks/node-ops/access-onchain-data/access-nodes/access-node-setup#step-4---start-your-node) for anyone who wants to run the node as a binary.